lib/attrsets.nix

```
@@ -333,6 +333,39 @@ rec {
       ) (attrNames set)
     );
 
+   /*
+    Like builtins.foldl' but for Attrsets.
+    Iterates over every nameValuePair in the given AttrSet.
+    The result can be appended to `acc` which gets finally returned.
+    Example
+      reduceAttrs
+      (acc: {name, value}:
```

```suggestion
      (acc: name: value:
```

All of the attrset-specific present the pair as a curried function call, so let's make it consistent.
Another benefit is that callers can choose their own names more easily.